src/test/java/com/squareup/javapoet/TypeSpecTest.java

```diff
@@ -839,7 +836,7 @@ public void interfaceDefaultMethods() throws Exception {
         + "\n"
         + "  com.squareup.donuts.Top externalTop;\n"
         + "\n"
-        + "  Bottom externalBottom;\n"
+        + "  com.squareup.donuts.Bottom externalBottom;\n"
```

Comment:
Import does not work as before here ... but the fall-back to fully-qualified class name is never wrong.